### ISSUE TITLE
Edit Page UI - utilize a tap to edit a page in project view 

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-core#445]
Currently in the project view, a user selects a page and then taps a pencil to open and edit a page. This has proven to be subpar. Instead, let's utilize a tap to open and edit a page. 
- Aligns with observed user behaivor
- Increases consistency between project Viewer and Editor
- Allows us to remove awkward selected page state
- Reduces steps and UI elements required

This change requires us to:
- remove delete action from project view
- add delete to page view more actions menu

**Comment by xmatthewx** [comment]
![page actions annotated](https://cloud.githubusercontent.com/assets/53411/9137192/163daae4-3cea-11e5-93d4-64e8c6b8ed8f.png)

:red_circle: **Note:** Disregard the neightbours as they appear on the right hand side of this image. In edit mode, the user will not see the neighbours.

R? @vazquez

**Comment by vazquez** [comment]
The zoom out button the right hand side is missing it's white circled background (minor but might be confusing). 

Other than that, R+!

Excited to see how this improves user's editing.

**Comment by xmatthewx** [comment]
Great. Thanks. Updated annotations above to include margin on divider. Removed zoom from comp since it's not in current build.

**Comment by vazquez** [comment]
:+1: :100:

**Comment by Pomax** [comment]
I really need to point out that removing user options from the immediate UI and hiding them in a hamburger is not the way towards user uptake. Articles like http://deep.design/the-hamburger-menu/ constantly remind us of this, and I would strongly urge us to not go down this road: keep our users in control by giving them directly usable options for the actions they want to take, and only put things that are more advanced in a "more options" menu (ideally without the "hamburger" or "kebab" icon). Creating and deleting pages is basic UI, not something to hide away.

**Comment by xmatthewx** [comment]
I hear that and I would pull "share" out from our kabob without hesitation. 

Delete page is not a commonly used feature given how easy it is to remove and add new elements on a page. It adds clutter and complexity to the primary role of the project view. We previously designed UI that put delete at the top corner of a page in page view, and could consider moving it there if tests show we need it. 

To edit a page, tests since May have consistently revealed that users attempt to open a page by tapping on it. When that doesn't work, they often try double tap before they hunt around for another option. Imagine if this approach (select a thumbnail, then tap an icon to open) was applied to thumbnails in a photo gallery or apps icons on your home screen? The project view currently runs counter to these common experiences.

**Comment by Pomax** [comment]
yeah, turning edit into an intuitive action like tap/double tap makes sense, but delete shouldn't be in a menu. The typical android behaviour there is tap-and-hold to get "advanced" options, so that might be a mode to explore as well.

**Comment by xmatthewx** [comment]
:+1: We've discussed using a tap-hold in project view for move page as well.

**Comment by Pomax** [comment]
yeah long tap opens up some good intuitive ui. long-tap and hold for additional actions, long tap and move around for.. well.. moving pages around =D
